AR-Top-MobileApp/Assets/Prefabs/RaycastText.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
```

Comment:
Do we need the first two using statements here? Just wondering if they are required.

Reply:
Strictly speaking they might not be, but they come with a LOT of functions C# and Mono Behavior depend on. Basically every script created from Unity automatically includes those two using statements.